arch/wec7/ua_architecture.h

```diff
@@ -0,0 +1,130 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information.
+ *
+ *    Copyright 2018 (c) Stephan
```

Comment:
Perhaps it would be best to describe which Stephan this wrote :)

Reply:
Jep, please include your full name, or your github username in parenthesis.